backend/src/db/repositories/PostRepository.ts

```diff
@@ -93,7 +95,7 @@ export default class PostRepository {
                      left join user_bookmarks b on (b.post_id = p.post_id and b.user_id = :for_user_id)
             where p.author_id = :user_id
                 ${filter ? ' and (p.source like :filter or p.title like :filter) ' : ''}
-            order by created_at desc
+            order by ${sorting === FeedSorting.postCommentedAt ? 'commented_at' : 'created_at'} desc
```

Comment:
bad readability - needed to reach to find `FeedSorting` declaration to find that this is enum. Also it will stop working as expected if more values will be added there. How about add a map with values:
```
const orderByMap: Record<keyof typeof FeedSorting, string> = {
  [FeedSorting.postCommentedAt]: 'commented_at',
  [FeedSorting.postCreatedAt]: 'created_at'
}
```

Reply:
We already use this approach in other places, refactoring it is outside of the scope of this PR.